### ISSUE TITLE
The loss suddenly becomes larger and the map becomes 0

### ISSUE THREAD

**Issue by yds5817** [AlexeyAB/darknet#6593]
Hello! The model I trained looked normal at the beginning, but then the loss would suddenly rise. What was going on？
![image](https://user-images.githubusercontent.com/46921972/91954429-606c1600-ed34-11ea-84ff-b9379e59ee83.png)

```
./darknet detector train ./cfg/wujie.data ./cfg/yolov4-custom.cfg ./yolov4.conv.137 -dont_show -mjpeg_port 9080 -map -gpus 0,1,2,3
 DEBUG=1
 CUDA-version: 9000 (11000), cuDNN: 7.1.4, CUDNN_HALF=1, GPU count: 4
 CUDNN_HALF=1
 OpenCV version: 3.4.10d
0,1,2,3
 Prepare additional network for mAP calculation...
 0 : compute_capability = 610, cudnn_half = 0, GPU: TITAN Xp
net.optimized_memory = 0
mini_batch = 1, batch = 32, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
```

**Comment by stephanecharette** [comment]
I recently had the same problem.  Was asking on the discord channel just the other day.  Never did figure it out.

![image](https://user-images.githubusercontent.com/5061352/91969611-3b8f8880-eccb-11ea-89bf-6a6a7a7eb981.png)

I think that project was yolov4-tiny-3l, but I'm no longer 100% certain.

**Comment by dsgh2** [comment]
Me too.
I solved this problem by decreasing learning rate a little bit.
But I am wondering why mAP only rises to a certain point and then goes to zero. It seemed to go well.

**Comment by zyayoung** [comment]
Same issue.
The sudden drop will occur near the same iter even if restart trainning from the last normal checkpoint.

**Comment by WANGCHAO1996** [comment]
![图片](https://user-images.githubusercontent.com/51087601/98440451-b776d880-2133-11eb-8771-bc6c315c8b82.png)


Hello, how to solve this problem, class will be 1, will be 0, IOU is very low..
